bleak/backends/bluezdbus/discovery.py

```diff
@@ -53,50 +53,178 @@ def _device_info(path, props):
         return None, None, None, None
 
 
-async def discover(timeout=5.0, loop=None, **kwargs):
-    """Discover nearby Bluetooth Low Energy devices.
+def _parse_device(path, props):
+    if not props:
+        logger.debug(
+            "Disregarding %s since no properties could be obtained." % path
+        )
+        return
```

Comment:
isn't it better to explicitly return  None , for these cases ? Does it make any difference?

Reply:
yeah, that was copied from the old code... But yes, I am a fan of explicit returns here as well...